File: towerRedo/Models/Comment.cs
namespace towerRedo.Models;

public class Comment : DbItem<int>
{
    public string CreatorId { get; set; }
    public int EventId { get; set; }
    public string Body { get; set; }
    public Boolean IsAttending { get; set; }
    public Account Creator { get; set; }
}

File: towerRedo.client/src/models/Comment.js
export class Comment {
    constructor(data = {}) {
        this.body = data.body
        this.id = data.id
        this.eventId = data.eventId
        this.isAttending = data.isAttending
        this.creatorId = data.creatorId
        this.creator = data.creator
    }
}